libyul/AsmPrinter.cpp

```diff
@@ -103,14 +103,15 @@ string AsmPrinter::operator()(FunctionalInstruction const& _functionalInstructio
 		boost::to_lower_copy(instructionInfo(_functionalInstruction.instruction).name) +
 		"(" +
 		boost::algorithm::join(
-			_functionalInstruction.arguments | boost::adaptors::transformed(boost::apply_visitor(*this)),
-			", ") +
+			_functionalInstruction.arguments | boost::adaptors::transformed([&](auto&& _node) { return std::visit(*this, _node); }),
```

Comment:
:(

Reply:
Can we maybe wrap that into a function? `make_visitor(*this)`?

Reply:
Also not a fan of the way this ended up.
Maybe @ekpyron @christianparpart have a better idea for this case?